src/code42cli/extensions.py

```diff
@@ -0,0 +1,5 @@
+from code42cli.click_ext.groups import ExtensionGroup
+from code42cli.main import CONTEXT_SETTINGS
+from code42cli.options import sdk_options  # noqa: F401
```

Comment:
wondering why are ignoring unused imports or suppressing warnings, `noqa...` statement? I need to read more on the usage of this statement, but the first thing that comes to my mind is will this suppress warnings only in context to this module or will it suppress all-over `code42cli`!!?

Reply:
Also can this be resolved by abstraction/design, breaking up the modules in such a way that we don't have to suppress any warnings or unused imports!

Reply:
I'm pretty sure the comment only suppresses it for the line being commented, I still got unused import style failures after this commit went in. 

I imported sdk_options here just for the convenience of script-writers, so they can just do:

```
from code42cli.extensions import script, sdk_options
```

Reply:
Another way to get around this would be to create a directory named `extensions` with an `__init__.py` importing things for convenience; it seems like that is ok to do because we do it other places to make importing things more convenient.

Reply:
Ah, good call. Making it a module would also allow us to better expand/organize things in the future.

Reply:
Huh, so the style cop still balks at unused imports even in the module __init__. I wonder why it doesn't do that in py42? I'm not seeing any config differences...

Reply:
nevermind. I found where we're ignoring it in the setup.cfg